### PR TITLE
Add settings to support capabilities

### PR DESCRIPTION
**Pull request by vetlek**
Linked to [ISAR-PR](https://github.com/equinor/isar/pull/241)

### CODE REVIEW

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: ["3.10"]
```
Comment (tsundvoll) [inline .github/workflows/pythonpackage.yml:17]: I disagree that we should require 3.10

**Review by tsundvoll — CHANGES_REQUESTED** [review]

File: src/isar_robot/config/settings.env [hunk 2]
```diff
@@ -0,0 +1 @@
+CAPABILITIES = '["take_thermal_image","take_image","drive_to_pose"]'
```
Comment (tsundvoll) [inline src/isar_robot/config/settings.env:1]:
```suggestion
CAPABILITIES = '["take_thermal_image", "take_image", "drive_to_pose"]'
```

**Review by tsundvoll — CHANGES_REQUESTED** [review]

File: .github/workflows/pythonpackage.yml (hunk 1 above)
Comment (vetlek) [reply .github/workflows/pythonpackage.yml:17]: Already an requirement in ISAR and needed for running the test

Comment (tsundvoll) [reply .github/workflows/pythonpackage.yml:17]: I see

**Review by tsundvoll — APPROVED** [review]

**Pull request merged**